src/app/components/pool-size-chart/pool-size-chart.component.scss

```diff
@@ -6,3 +6,12 @@
 #stats {
   height: 100%;
 }
+
+.pool-size-chart-controls {
+  text-align: right;
+}
+
+.pool-size-chart-controls button {
+  margin-left: 9px;
+  width: 110px;
```

Comment:
Best to try to avoid using width and use paddings, some users may prefer larger text for example.